.idea/codeStyleSettings.xml

```diff
@@ -289,6 +289,9 @@
         <JavaCodeStyleSettings>
           <option name="DO_NOT_WRAP_AFTER_SINGLE_ANNOTATION" value="true" />
         </JavaCodeStyleSettings>
+        <MarkdownNavigatorCodeStyleSettings>
```

Comment:
Looks like it was added by the markdown plugin that I am using with Android Studio. Need to remove this.